### ISSUE TITLE
error deleting or resending user invitations

### ISSUE THREAD

**Issue by optitsolutions** [humhub/massuserimport-issues#15]
### What steps will reproduce the problem?
After user registers on login page as new user I resend invitation or delete the invitation on "administration"->"user" ->"invite users"

### What is the expected result?
He gets invitation mail (ok). humhub admin can resend or delete pending invitation.

### What do you get instead?
humhub throws an error ("Invalid request method!") if I delete or resend the invitation.
(without error log entry)

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.14.4
| PHP version      | 8.0.11
| Operating system | win 11 pro

**Comment by ArchBlood** [comment]
Update to a newer version manually or use the updater module.

https://download.humhub.com/

**Comment by optitsolutions** [comment]
Sorry, edit above.

It's the newest stable version.

**Comment by optitsolutions** [comment]
the error appears with new (today) and older invitations (weeks and months ago).

**Comment by ArchBlood** [comment]
Could you provide the error stack from ACP > Information > Logging, only provide the one labeled `Error`

**Comment by optitsolutions** [comment]
no log entry appears ;-(

**Comment by ArchBlood** [comment]
Check `/protected/runtime/log/app.logs` normally this will generate the error even if it doesn't show from the ACP logging.

**Comment by optitsolutions** [comment]
I just forced the error again - no (new) entries in the app.logs file.
I rebuilt the search index and cleared the cache already.
Any other ideas?

**Comment by ArchBlood** [comment]
Check the browser console for script errors when the action is done, also do the same on another new account to see if it repeats the browser console error if any.

**Comment by optitsolutions** [comment]
thank you, good idea.
Here we are:
Anfrage-URL:
https://xxx.tld/index.php?r=massuserimport%2Finvite%2Fdelete&id=9&doit=2
Anfragemethode:
GET
Statuscode:
405 Method Not Allowed (von service worker)

It refers to the Mass User Import module, but the edited records are not created with the module, but via a separate registration (login page)

**Comment by optitsolutions** [comment]
@luke- maybe a mass user import module issue?

**Comment by yurabakhtin** [comment]
@luke- I think the bugs were fixed in the PR https://github.com/humhub/massuserimport/pull/22, but it is not released yet.

**Comment by luke-** [comment]
@yurabakhtin Thanks, can you please check: https://github.com/humhub/massuserimport/issues/23

**Comment by Semir1212** [comment]
The invitation/accounts seem to be stilll there after defletion. Could you take a look into that @yurabakhtin

**Comment by yurabakhtin** [comment]
> The invitation/accounts seem to be stilll there after defletion. Could you take a look into that @yurabakhtin

@Semir1212 Could you please check I tested correctly the case you mean?

![deleted-invite](https://github.com/humhub/massuserimport-issues/assets/6995524/127a2e86-6759-47e5-838e-697761975df5)

**Comment by optitsolutions** [comment]
It works fine here since the last update.

**Comment by Semir1212** [comment]
@yurabakhtin 

I have received another message from the same user that it does not work. There were still entries in his installation that I created during testing after the last update. 

When I tried to delete the invitation, the same error appeared as before. I then changed the e-mail address in the CSV and received the error that the user already exists.

After creating a completely new CSV and adding a user, I was able to delete the initial entry as well as the new one. It seems to be a problem with the CSV, so let's close this for now.
